packages/core/src/cells/drilldown-cell.tsx

```diff
@@ -87,9 +87,8 @@ function getAndCacheDrilldownBorder(
 
     drilldownCache[key] = canvas;
 
-    const trueRounding = Math.min(rounding, targetWidth / 2, targetHeight / 2);
```

Comment:
I removed this because the overlay editor also just used a static value (6px), but I can also try to figure out how to keep this logic in.